src/internal/assets/geoPushpin/bucket-title-color.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="16" height="10" xmlns="http://www.w3.org/2000/svg">
```

Comment:
do we need 4 same icons?

Reply:
I think so, this one is provided by Designer team

Reply:
I know, but all 4 buckets have the same icon, so I dont see reason why dont reuse one file for all 4 buckets

Reply:
I thought you were talking about `<svg width="16" height="10"`, not visual shape 
What a nice catch. Fixed
Thanks